common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -440,6 +440,11 @@ workflow_state = ? ` +
 		`and visibility_ts = ? ` +
 		`and task_id = ?`
 
+	templateGetExecutionsQuery = `SELECT domain_id, workflow_id, run_id, current_run_id, workflow_state ` +
```

Comment:
Can we reuse templateListWorkflowExecutionQuery?

Reply:
The templateListWorkflowExecutionQuery selects unnecessary stuff while we need to keep it as lite as possible for the scanner.

Reply:
My personal preference is the simplicity over efficiency here as this triggers from a background job. But I don't have a strong opinion.